src/app/hearing/services/hearing.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { HearingService } from './hearing.service';
+
+describe('HearingService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [HearingService]
+    });
+  });
+
+  it('should be created', inject([HearingService], (service: HearingService) => {
```

Comment:
this spec is a bit poor. It would be good to have some unit test that hit proper URL and return proper values